plaid/README.md

```diff
@@ -22,18 +22,36 @@ Here are some insights that can be drawn from your Plaid dashboard:
 ### Configuration
 
 Configure the Datadog endpoint to forward Plaid logs to Datadog.
-1. Navigate to Plaid.
-2. Add your Plaid credentials.
+1. Log in to [Plaid Dashboard](https://dashboard.plaid.com/)
+2. Navigate to the **Developers** section in the left pane.
+3. Extend the drop-down menu and click on **Keys** 
+4. Obtain the client_id and Secret.
+
+#### To obtain Access Token, follow these steps:
+   1. **Get institution_id from Plaid**:  
+      Hit Plaid API **/institutions/get** endpoint to obtain institution_id. Reference [link](https://plaid.com/docs/api/institutions/#institutionsget) 
+   2. **Create a Public Token**:  
+      You will need to create a public token. Use the institution_id that you retrieved from Step 1 and hit **/public_token/create** endpoint. Reference  [link](https://plaid.com/docs/api/sandbox/#sandboxpublic_tokencreate) 
```

Comment:
```suggestion
1. [**Retrieve the `institution_id` from Plaid**](https://plaid.com/docs/api/institutions/#institutionsget) :  
    Use the Plaid API's **/institutions/get** endpoint to fetch the `institution_id`.
2. [**Create a Public Token**](https://plaid.com/docs/api/sandbox/#sandboxpublic_tokencreate):  
    Create a public token by using the `institution_id` that you retrieved from Step 1 and hit **/public_token/create** endpoint.
```

Reply:
Can you move these links to the bottom of the page. these should be formatted like this:
[**Retrieve the `institution_id` from Plaid**][#] - replace `#` with the link number.